src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -893,15 +893,35 @@ def test_process_array_declarations():
     reader = FortranStringReader("integer :: l5(2), l6(3)")
     fparser2spec = Specification_Part(reader).content[0]
     processor.process_declarations(fake_parent, [fparser2spec], [])
-    assert fake_parent.symbol_table.lookup("l5").datatype.shape == [2]
-    assert fake_parent.symbol_table.lookup("l6").datatype.shape == [3]
+    l5_datatype = fake_parent.symbol_table.lookup("l5").datatype
+    assert len(l5_datatype.shape) == 1
+    assert isinstance(l5_datatype.shape[0], Literal)
+    assert l5_datatype.shape[0].value == '2'
+    assert l5_datatype.shape[0].datatype.intrinsic == ScalarType.Intrinsic.INTEGER
```

Comment:
Watch out the pycodestyle 80 character limits (there are some introduced in this file and `src/psyclone/tests/psyir/nodes/literal_test.py`)

Reply:
Oops. I really should have gone through pycodestyle and pylint issues. Apologies. I'll double check files before giving this back for next review.